src/components/home-banner/home-banner.js

```diff
@@ -1,20 +1,31 @@
 import './home-banner.scss'
-import bannerOverlay from '../../assets/home-banner-overlay.png'
+import proJects from '../../assets/projects.png'
```

Comment:
Our imports use Pascal Case and the casing switches on the word not the syllable so:
Projects, Mobility, Parking, etc. If you had "mobility projects" it would be MobilityProjects

Reply:
Good catch, pretty sure that's a typo